.github/workflows/AtlasReferenceAppTest.yml

```diff
@@ -0,0 +1,49 @@
+name: Testing Atlas Reference App
```

Comment:
We don’t need this file anymore as now it is following the same approach as widgets e2e

Reply:
Indeed, I removed this file as suggested.